resources/public/locales/en/translation.json

```diff
@@ -1,290 +1,293 @@
 {
```

Comment:
Why did this whole file change? Same question on the Spanish `translation.json` file as well